README.md

```diff
@@ -1,5 +1,15 @@
 # boats.py
-The official discord.boats API wrapper for Python
+The official Python Library for the [discord.boats](https://discord.boats) API
+
+# Installation
+## Install via pip (Recommended)
+```pip install boats.py```
+## Install from source
+```
+git clone https://github.com/DiscordBoats/boats.py
+cd boats.py
+pip install -r setup.txt
```

Comment:
That's not how it works. It is python setup.py install

Reply:
I... I meant to put `.py` but then I got distracted sorry...

Reply:
Ok, Thanks! I forgot to put `.py` but I don't really work with setup files often so thanks :)